### PR TITLE
BusinessDashboard Theme | <Badge> - Changed spacings and colors

### PR DESCRIPTION
**Pull request by shelly-goldblit**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6985/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6985

File: packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css [hunk 1]
```diff
@@ -19,14 +19,18 @@
 
 .root {
   border-radius: 2px;
-  padding: 2px 8px;
-  line-height: 16px;
+  padding: 4px 8px;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css:22]: we have spacing import for SP1 / SP2

File: packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css [hunk 2]
```diff
@@ -19,14 +19,18 @@
...
   height: 20px;
 }
 
+.root span {
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css:26]: All spans should have it? or do you have class name for what you need?

File: packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css [hunk 3]
```diff
@@ -19,14 +19,18 @@
...
+  line-height: 1.2;
+}
+
 .root:size(small) {
   height: value(smallHeight);
   padding: value(smallPadding);
+  padding: 2px 8px;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css:33]: 2px is correct?
We better use spacing imports

File: packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css [hunk 4]
```diff
@@ -38,3 +42,37 @@
   border-color: value(D20);
   background-color: value(D20);
 }
+
+.root:skin(success) {
+  border-color: value(G20);
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css:47]: Do we need the border?

File: packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css [hunk 5]
```diff
@@ -38,3 +42,37 @@
...
+}
+
+.root:skin(neutralStandard) {
+  border-color: value(B40);
+  background-color: value(B40);
+}
+.root:skin(neutralDanger) {
+  border-color: value(R40);
+  background-color: value(R40);
+}
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css:78]: Do we support all those skins?

File: packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css [hunk 6]
```diff
@@ -39,12 +39,17 @@
   /* Success */
   G10: #51B77B;
   G20: #51B77B;
-  G40: #C8E8D6;
+  G40: #CDF2CF;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css:42]: Synced with Ayelet?

File: packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css (hunk 2 above)
Comment (shelly-goldblit) [reply packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css:26]: all spans inside badge

File: packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css (hunk 5 above)
Comment (shelly-goldblit) [reply packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css:78]: Please look at the spec

File: packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css (hunk 6 above)
Comment (shelly-goldblit) [reply packages/wix-style-react/src/Themes/businessDashboard/foundations/colors.st.css:42]: https://jira.wixpress.com/browse/OSB-456

File: packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css (hunk 4 above)
Comment (shelly-goldblit) [reply packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css:47]: why?

File: packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css (hunk 3 above)
Comment (shelly-goldblit) [reply packages/wix-style-react/src/Themes/businessDashboard/components/Badge.st.css:33]: There is no spacing import for 2px. 2px is what's defined in the spec

**Review by adicicurel — APPROVED** [review]

**Pull request merged**